okio-testing/src/commonMain/kotlin/okio/FakeFilesystem.kt

```diff
@@ -133,7 +133,7 @@ class FakeFilesystem(
     val element = requireDirectory(canonicalPath)
 
     element.access(now = clock.now())
-    return elements.keys.filter { it.parent == canonicalPath }
+    return elements.keys.filter { it.parent == canonicalPath }.sorted()
```

Comment:
Might be worth ensuring that allPaths is also predictable for tests.

Reply:
ooooh good call.